packages/in_app_purchase/in_app_purchase_storekit/example/ios/RunnerTests/TranslatorTests.m

```diff
@@ -390,4 +390,21 @@ - (void)testSKPaymentDiscountFromMapMissingTimestamp {
   }
 }
 
+- (void)testSKPaymentDiscountFromMapOverflowingTimestamp {
+  if (@available(iOS 12.2, *)) {
+    NSDictionary *discountMap = @{
+      @"identifier" : @"payment_discount_identifier",
+      @"keyIdentifier" : @"payment_discount_key_identifier",
+      @"nonce" : @"d18981e0-9003-4365-98a2-4b90e3b62c52",
+      @"signature" : @"this is a encrypted signature",
+      @"timestamp" : @1665044583595,  // timestamp 2022 Oct
+    };
+    NSString *error = nil;
+    SKPaymentDiscount *paymentDiscount =
+        [FIAObjectTranslator getSKPaymentDiscountFromMap:discountMap withError:&error];
+    XCTAssertNil(error);
+    XCTAssertNotNil(paymentDiscount);
```

Comment:
Can we also check members in the `paymentDiscount` have the same value with `discountMap`. In the case of this Pr, especially `timestamp`

Reply:
Done.